components/Cards/ResourceCard/ResourceSkeletonCard.tsx

```diff
@@ -1,14 +1,17 @@
-import { number } from 'prop-types';
 import { SKELETON_CARD } from '../../../common/constants/testIDs';
 import styles from './ResourceSkeletonCard.module.css';
 
-ResourceSkeletonCard.propTypes = {
-  numberOfSkeletons: number.isRequired,
+export type ResourceSkeletonCardPropsType = {
+  /**
+   * Sets the number of UI "skeletons" to render.
+   */
+  numberOfSkeletons: number;
 };
 
-function ResourceSkeletonCard({ numberOfSkeletons }) {
+function ResourceSkeletonCard({ numberOfSkeletons }: ResourceSkeletonCardPropsType) {
   return (
     <div className={styles.resourcesCardWrapper}>
+      {/** @ts-expect-error */}
       {[...Array(numberOfSkeletons)].map((skeleton, index) => (
```

Comment:
```suggestion
      {/** @ts-expect-error */}
      {[...Array(numberOfSkeletons)].map((_skeleton, index) => (
```

does the type arrow go away if you do this?

Reply:
Yea, that worked. Can you explain that, or let me know something I can search to read up on what happened there?

<img width="711" alt="Screenshot 2024-03-03 at 7 50 53 PM" src="https://github.com/OperationCode/front-end/assets/46408716/75b30605-0c93-416b-ab62-29a36a5884af">

Reply:
So, TypeScript complains about unused vars from a compiler standpoint beside the linter.

We have a lint rule that matches a TypeScript paradigm where prefixing unused variables with a _ means we know they're unused, but it's necessary.

The _ prefix is a thing in other languages for the same purpose too.